### DISCUSSION TITLE
commitBody is ignored for onboarding branch commit

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#25926] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

How can I make those commits to be signed-off?

Thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Signing off commits and GPG signing of commits are different things, see https://medium.com/@MarkEmeis/git-commit-signoff-vs-signing-9f37ee272b14

Which are you referring to? Please update the title/question to clarify

**Reply by lmilbaum** [reply]
@rarkins I don't think this issue metions GPG or maybe I am not following your response. I just want to the onboarding commits to be signed-off.

**Reply by rarkins** [reply]
You don't mention if you tried https://docs.renovatebot.com/presets-default/#gitsignoff or not

**Reply by lmilbaum** [reply]
I actually use this preset but it applies only for PRs which are generated by Renovate after the onboarding PR is merged

**Reply by rarkins** [reply]
Can you update the title and description to match?

**Reply by lmilbaum** [reply]
Done

**Comment by rarkins** [comment]
I think the bug here may be generalized as "commitBody is ignored for onboarding branch commit"

**Reply by lmilbaum** [reply]
At what stage the commitBody is formed? AFAIK, the preset is not known to Renovate at the time of generating the onboarding PR.

**Comment by rarkins** [comment, marked as answer]
Converted to bug issue: #25943
